stable/mariadb/values.yaml

```diff
@@ -7,6 +7,11 @@
 #   imagePullSecrets:
 #     - myRegistryKeySecretName
 
+## Use an alternate scheduler, e.g. "stork".
+## ref: https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/
+##
+# schedulerName:
```

Comment:
Apply the same changes in the `values-production.yaml` too

Reply:
Done.